lib/src/view/menu.dart

```diff
@@ -583,6 +585,15 @@ Shows grid coordinates in the side view under the helix index.'''
         ..onChange = ((_) => props.dispatch(actions.ShowGridCoordinatesSideViewSet(
             show_grid_coordinates_side_view: !props.show_grid_coordinates_side_view)))
         ..key = 'show-grid-coordinates-side-view')(),
+      (MenuBoolean()
+        ..value = props.show_loopout_length_main_view
+        ..display = 'Show loopout length in main view'
+        ..tooltip = '''\
+Loopout length is anchored to loopout when selected in main view.'''
```

Comment:
I'd phrase this "When selected, the length of each loopout is displayed next to it."